website/data/consul-on-kubernetes.js

```diff
@@ -0,0 +1,131 @@
+export const features = [
+  {
+    title: 'Multi-Platform',
+    subtitle:
+      'Support both Kubernetes and non-Kubernetes workloads on any runtime',
+    infoSections: [
+      {
+        heading: 'Why it matters',
+        content: (
+          <p>
+            You can connect almost any application to any runtime. Consul
+            supports virtual machines and containers across just about any
+            platform.
+          </p>
+        ),
+      },
+      {
+        heading: 'Features',
+        content: (
+          <ul>
+            <li>Run thousands of nodes with low latency</li>
+            <li>Support any Kubernetes distribution</li>
+            <li>Work across Kubernetes & non-Kubernetes Environments</li>
+          </ul>
+        ),
+      },
+    ],
+    cta: {
+      text: 'Try It Now',
+      url: '#TODO',
+    },
+    image: '/img/consul-k8/features/multi-platform.png',
```

Comment:
Let's switch all these PNGs to SVGs.  LMK if you need help exporting them!

Reply:
Additionally, could we house these images rather than in the public directory, in the page's folder maybe in a subfolder called `image` or something.  Strengthens encapsulation of the page logic!